src/utils.js

```diff
@@ -223,6 +232,8 @@ export const setVerboseMode = () => (process.env[VERBOSE_FLAG] = 'TRUE');
 
 export const setSwitchOff = () => (process.env['LUMIGO_SWITCH_OFF'] = 'TRUE');
 
+export const isSwitchOff = () => validateEnvVar('LUMIGO_SWITCH_OFF');
```

Comment:
```suggestion
export const isSwitchOffEnvVar = () => validateEnvVar('LUMIGO_SWITCH_OFF');
```
Because there is already a function called `isSwitchedOff` and it's a bit confusing.